scripts/constraints.txt

```diff
@@ -1,6 +1,6 @@
 #
-# This file is autogenerated by pip-compile with python 3.10
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.9
+# by the following command:
```

Comment:
There is so Python version mismatch here. I'm not sure if we can change this. Maintainers complain about this.

Reply:
There are many changes in this file and it doesn't seem related to the changes in the mbed requirements file.

Reply:
I was quite surprised about this as well!

Reply:
fixed